src/views/design/ScriptsOverview/index.tsx

```diff
@@ -195,17 +203,50 @@ const ScriptsOverview = withStyles(styles)(
                                     </Box>
                                     {checkAuthorityGeneral(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE)
```

Comment:
the 'load OpenAPI' button should appear if you have CONNECTIONS_WRITE or COMPONENTS_WRITE privilege.
The 'add button " button should appear if you have SCRIPTS_WRITE privilige (as is situation)